### ISSUE TITLE
Brain Tumor MRI Classification

### ISSUE THREAD

**Issue by theiturhs** [SrijanShovit/HealthLearning#8]
Being a GSSOC'24 contributor, I am looking forward to proceed with "Brain Tumor MRI Classification" focusing on MRI images. The dataset will undergo preprocessing steps such as cleaning (if required, suppose the image data contains some salt and pepper noise, it can be removed using Median filter), and feature extraction. For modeling, I will utilize convolutional neural networks (CNNs), well-suited for image classification tasks.

Thanks!

**Comment by Basma2423** [comment]
I worked on a BRAIN-MRI dataset before
Can you assign it to me? @SrijanShovit 

Full name: **Basma Mahmoud**
GitHub Profile Link: [Basma2423](https://github.com/Basma2423)
Email ID: mayarbasma2423@gmail.com
Approach for this Project: Multiple Deep Learning pre-trained models + CNN
What is your participant role? (Mention the Open Source program): **GSSoC-2024** participant

Can you add the label for **GSSoC**, please?
Thanks.

**Comment by HarshaliD** [comment]
Hi , I'm excited to contribute to this project. Could you please assign me? Looking forward to getting started! @SrijanShovit 

Full name: Harshali Devi
GitHub Profile Link: [HarshaliD](https://github.com/HarshaliD)
Email ID: harshalidevi@gmail.com
Approach for this Project: Trying various CNN architectures like ResNet50 and VGG-16 model using Tensorflow and Keras.
What is your participant role? : GSSoC-2024 participant

**Comment by SrijanShovit** [comment]
@theiturhs Please proceed: 
Step 1:

1. Explore classes and their respective sizes in dataset and perform necessary augmentation.
2. Decide upon the number of samples nearly for each class and relevant strategy for noise removal, augmentation(be careful about this).
3. Upload modified dataset on kaggle(easy to work with later).

These all steps make us ready to start working on a proper dataset. Next steps will have separate issues.

**Comment by theiturhs** [comment]
Thank you @SrijanShovit for providing this to me. I will proceed with provided Step 1.

**Comment by Pranali3103** [comment]
Hi @theiturhs I am GSSOC'24 contributor , please assign me this issue .

**Comment by SrijanShovit** [comment]
> Hi @theiturhs I am GSSOC'24 contributor , please assign me this issue .

Already taken up, you can search some other disease's dataset and come up with an idea.
